README.markdown

```diff
@@ -49,6 +49,35 @@ The Hypothesis LMS app is written for python 3 and uses Node.js and `yarn` for m
     $ source bin/activate
     ```
 
+1. Create a client_credentials auth client in h.
```

Comment:
May I suggest adding a tad more "why" to this statement? Like: "Create credentials so that the LMS app can create `h` users?" (The specifics of it being `auth_client`/`client_credentials` is well described within the instructions themselves).

Reply:
Done!